### ISSUE TITLE
Wrong directory for python scripts in documentation?

### ISSUE THREAD

**Issue by mgavioli** [fontforge/fontforge#4448]
**Context**: FontForge 2020-03-14 AppImage on Linux Mint 19.3 (<= Ubuntu 18.04)

In the https://fontforge.org/docs/scripting/python/fontforge.html documentation page, it says:

“When FontForge starts (if it’s a FontForge with python) it will look at the directories `$(PREFIX)/share/fontforge/python` and `~/.FontForge/python` and attempt to run all files in those directories which end in ".py"...”

On my machine, after several trials and errors, I discovered that the user directory for python scripts is actually `~/.config/fontforge/python` (which was already there, I hadn‘t to create it).

Is it an Ubuntu-specific detail or is the documentation out-of-date?

**Comment by jtanx** [comment]
Just guessing but it might be some xdg thing

**Comment by ctrlcctrlv** [comment]
Yeah, I was thinking the same, probably `$XDG_HOME` moved. Really, that should always have been the variable in the docs, not `$PREFIX`.
